redbot/core/core_commands.py

```diff
@@ -2357,16 +2358,16 @@ async def command_disabledmsg(self, ctx: commands.Context, *, message: str = "")
     @commands.group(name="autoimmune")
     async def autoimmune_group(self, ctx: commands.Context):
         """
-        Server settings for immunity from automated actions
+        Server settings for immunity from automated actions.
         """
         pass
 
     @autoimmune_group.command(name="list")
     async def autoimmune_list(self, ctx: commands.Context):
         """
-        Get's the current members and roles
+        Gets the current members and roles.
 
-        configured for automatic moderation action immunity
+        Configured for automatic moderation action immunity.
```

Comment:
This is one string here: "Gets the current members and roles configured for automatic moderation action immunity."